ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -2689,6 +2693,27 @@ - (void)getCurrentPage:(FlutterResult)flutterResult {
     flutterResult([NSNumber numberWithInt:documentController.pdfViewCtrl.currentPage]);
 }
 
+- (void)getZoom:(FlutterResult)flutterResult {
+    PTDocumentController *documentController = [self getDocumentController];
+    double zoom = [documentController.pdfViewCtrl GetZoom];
```

Comment:
I noticed the RN version returns:
`pdfViewCtrl.zoom * pdfViewCtrl.zoomScale`
on iOS at least.
Should it be the same here?

Reply:
this works but I see that is a lot shorter

Reply:
Hm not sure what you mean by shorter?

Reply:
maybe not